public/app/plugins/panel/trend/module.tsx

```diff
@@ -10,10 +10,10 @@ import { TrendSuggestionsSupplier } from './suggestions';
 export const plugin = new PanelPlugin<Options, FieldConfig>(TrendPanel)
   .useFieldConfig(getGraphFieldConfig(defaultGraphConfig, false))
   .setPanelOptions((builder) => {
-    const category = ['X Axis'];
+    const category = ['X axis'];
```

Comment:
hm changing the category casing may cause a change to the save model / potentially break things 🤔 the other changes should be just cosmetic though / not impact the save model

Reply:
That could be the wrong place to make that change. Goal was to correct the casing of Axis there. Where would be the correct place to do that?

Reply:
@nmarrs, @staton-hyse11 I think this one is ok too, it seems to be used only in the options UI, the save model is saving just `xField`.